http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -147,7 +147,9 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
           stagingContainerName.value,
           stagingContainerResourceId.value.toString,
           params.workspaceName,
-          wsStorageContainerUrl
+          wsStorageContainerUrl,
+          "https://leonardo.dsde-dev.broadinstitute.org", // ServiceHost field needs the leonardo URL to hit when calling dateAccessed - how do we get this value?
```

Comment:
@rtitle @giventocode  what is the real source of this URL? What should go here?

Reply:
Done, hopefully, here: https://github.com/broadinstitute/firecloud-develop/pull/3269

Reply:
You will need to add this field in `leo.conf` and an env var in `local.env` to be compatible with the independent release changes

Reply:
I merged the firecloud-develop PR above, without it we won't be able to test these changes